README.md

```diff
@@ -59,6 +59,40 @@ Please note:
 * Batch sizes specified in prototxt files are per device.
 * Memory layers should not be shared among GPUs, and thus "share_in_parallel: false" is required for layer configuration.
 
+## Building for Spark 2.X
+To Build for Spark 2.X  please include  
+
+* mvn -Dspark2
+
+**Spark2 Pre-release note**  You will need to do the following:
+- in the Spark git clone :
+
+-      mvn **install** \<other spark options\>
+
+Then:  update the CaffeOnSpark caffe-grid/pom.xml in the "spark2" profile section:  set the correct spark (e.g. **2.0.0-SNAPSHOT**),scala (e.g. **2.11.7**)), and hadoop (e.g. **2.7.1**)) versions:
+
+```
+  <profiles>
```

Comment:
So I have updated the pom.xml  Here is the updated info

<img width="751" alt="screen shot 2016-06-20 at 10 43 34 pm" src="https://cloud.githubusercontent.com/assets/410769/16219085/828e7090-3738-11e6-8fb9-cef32bd9a327.png">